src/main/webapp/view/transcript/transcript-area.js

```diff
@@ -77,6 +73,23 @@ export default class TranscriptArea {
     });
   }
 
+  /**
+   * Returns `transcriptContainer`.
```

Comment:
Here I don't know what `transcriptContainer` is since it is an internal function names. 

Maybe just say "Returns container containing transcript."

Reply:
updated, thanks!